dist/schema.json

```diff
@@ -35269,7 +35269,7 @@
           "requires_review"
         ],
         "externalDocs": {
-          "url": "https://docs.joinmastodon.org/entities/Admin::Tag/",
+          "url": "https://docs.joinmastodon.org/entities/Admin_Tag/",
```

Comment:
Keep track of the file entities are parsed from and use those in the URL. For example `Admin::Tag` is defined in the `content/en/entities/Tag.md` file so it should link to `https://docs.joinmastodon.org/entities/Tag/#admin`